lib/wasi/src/runtime/host_ws.rs

```diff
@@ -0,0 +1,90 @@
+// TODO: move to more appropriate submodule
```

Comment:
Address TODO

Reply:
This file wasn't actually included in the module tree!

I deleted it.

I preserved the code in `sys-utils/src/tungstenite_websocket`, but with a FIXME.
Should probably just be removed and replaced with WAI based websocket bindings in the future.